### DISCUSSION TITLE
Warn about duplicate links

### DISCUSSION THREAD

**Discussion by etherbob** [ruby/rubygems#6654] [Ideas]
If https://github.com/rubygems/rubygems.org/pull/3755 gets merged duplicate links in the gemspec will get hidden on rubygems.org and only the first in the list defined in models/links.rb will be displayed. @simi pointed out we could raise a warning to gem authors at build time by adding a warning to [specification_policy.rb](https://github.com/rubygems/rubygems/blob/ffad5e639fb21131b9285c4cb3f4798e53687ee9/lib/rubygems/specification_policy.rb)

I thought I would check in here and see if that's a desired feature before I dug too far into implementation.

**Comment by hsbt** [comment]
This was worth doing and it landed in https://github.com/ruby/rubygems/pull/7213. `gem build` now warns when the same URI is given for more than one metadata link key, and says that only the first will be shown on rubygems.org. It is in the current release.
